CHANGES.md

```diff
@@ -7,6 +7,7 @@
 - [NEW] Add `ComplexKey.addHighSentinel()` to allow matching of all values as part of a complex key
   range.
 - [NEW] Support IAM authentication in replication documents.
+- [NEW] Add support `stable` and `update` parameters in views.
```

Comment:
`Add support for` or `Support...`

Reply:
Also should add a `DEPRECATED` for the `stale`

Reply:
Done in 54ab529